backend/src/core/middlewares/campaign.middleware.ts

```diff
@@ -28,6 +28,12 @@ const canEditCampaign = async (
       CampaignService.hasJobInProgress(+campaignId),
       UploadService.getCsvStatus(+campaignId),
     ])
+    logger.info({
+      message: 'Successfully created new campaign',
+      hasJob,
+      csvStatus,
+      action: 'createCampaign',
```

Comment:
I think the `action` should be aligned with the method name, i.e. should be `canEditCampaign` rather than `createCampaign`.

think this is true for the other `logger.info`s too

Reply:
fixed w next push